src/schema/v2/me/update_me_mutation.ts

```diff
@@ -190,18 +199,29 @@ export default mutationWithClientMutationId<any, any, ResolverContext>({
     },
     me: Me,
   },
-  mutateAndGetPayload: async (args, { updateMeLoader }) => {
+  mutateAndGetPayload: async (
+    { iconUrl, ...args },
+    { updateMeLoader, updateCollectorProfileIconLoader }
+  ) => {
     // snake_case keys for Gravity (keys are the same otherwise)
     const user = Object.keys(args).reduce(
       (acc, key) => ({ ...acc, [snakeCase(key)]: args[key] }),
       {}
     )
 
-    if (!updateMeLoader) {
-      throw new Error("No updateMeLoader loader found in root values")
+    if (!updateMeLoader || !updateCollectorProfileIconLoader) {
+      throw new Error(
+        "No updateMeLoader or updateMeIconLoader loader found in root values"
```

Comment:
**suggestion:** The only way this would occur is when the user is not signed in so I suggest we update this to be
```suggestion
        "You need to be signed in to perform this action"
```